input/pagecontent/terminology.md

```diff
@@ -37,7 +37,7 @@ Many value sets used in this guide are defined in the base FHIR specification, [
 |[RequestIntent](https://hl7.org/fhir/R4/valueset-request-intent.html)|[AU eRequesting ServiceRequest](StructureDefinition-au-erequesting-servicerequest.html)|FHIR|
 |[RequestStatus](https://hl7.org/fhir/R4/valueset-request-status.html)|[AU eRequesting ServiceRequest](StructureDefinition-au-erequesting-servicerequest.html)|FHIR|
 |[AU eRequesting Task Status](ValueSet-au-erequesting-task-status.html)|[AU eRequesting Task](StructureDefinition-au-erequesting-task.html)|AU eRequesting|
-|[AU eRequesting Coverage Type Self-pay](ValueSet-au-erequesting-coverage-self-pay.html)||AU eRequesting|
+|[AU eRequesting Coverage Type Self-Pay](ValueSet-au-erequesting-coverage-self-pay.html)||AU eRequesting|
```

Comment:
This still does not align with the value set title: AU eRequesting Coverage Type _and_ Self-Pay but it may require other changes once the value set naming conventions are addressed. 

Please update all instances (here, and the Code systems table).